src/app/signin/signin.component.html

```diff
@@ -10,17 +10,19 @@ <h1>Connexion</h1>
       <mat-form-field class="example-full-width">
         <input matInput placeholder="Mot de passe" type="password" formControlName="password">
       </mat-form-field>
-
       <button mat-raised-button color="primary" type="submit"
               [disabled]="signinForm.invalid">Connexion
       </button>
-      &nbsp; &nbsp;
+      &nbsp;
```

Comment:
un détail pour la prochaine fois,  il n'est pas conseiller d'utiliser l'espace &nbsp; pour distancer des elements html.
à la place, il faut le faire avec du css, par exemple
```suggestion
a {
      margin-left: 10px
}
```